Fabric.Identity.API/Models/ModelExtensions.cs

```diff
@@ -120,8 +122,8 @@ public static Client ToClientViewModel(this IS4.Client client)
                 PostLogoutRedirectUris = client.PostLogoutRedirectUris,
                 EnableLocalLogin = client.EnableLocalLogin,
                 IdentityProviderRestrictions = client.IdentityProviderRestrictions,
-                LogoutUri = client.LogoutUri,
-                LogoutSessionRequired = client.LogoutSessionRequired,
+                LogoutUri = client.FrontChannelLogoutUri ?? client.BackChannelLogoutUri,

```

Comment:
So based on this statement, we are using either implicit flow or authorization code flow, but the Client allowedGrantType is only Implicit.